websites/R/Roblox Forum/presence.ts

```diff
@@ -0,0 +1,61 @@
+const presence = new Presence({
+	clientId: "993929577099448481",
+});
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "ig-mal",
+			smallImageKey: "small",
+			smallImageText: "Swooosh!",
+		},
+		date = Math.floor(Date.now() / 1000);
```

Comment:
For repo consistency
```suggestion
		browsingTimestamp = Math.floor(Date.now() / 1000);
```

Reply:
Okay, I'll be doing this once I'm done eating